CHANGELOG.md

```diff
@@ -4,6 +4,13 @@ Todos los cambios notables a este proyecto serán documentados en este archivo.
 El formato está basado en [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 y este proyecto adhiere a [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [1.1.2] - 2021-11-17
+### Fixed
+- Se actualiza versión del SDK de PHP a la última versión.
```

Comment:
I think it is better to indicate the version of the PHP SDK, in the future it will not be known in a simple way which was the last of that release.